contracts/implementation/PoolFactory.sol

```diff
@@ -161,7 +161,7 @@ contract PoolFactory is IPoolFactory, Ownable {
         feeReceiver = _feeReceiver;
     }
 
-    function setFee(bytes16 _fee) external override onlyOwner {
+    function setFee(uint256 _fee) external override onlyOwner {
```

Comment:
Might be worth a comment, but this fee should now be a NON WAD value - since on the constructor of the pool it converts it to a WAD.